sympy/series/tests/test_fourier.py

```diff
@@ -65,6 +65,17 @@ def test_FourierSeries_2():
                             4*cos(pi*x / 2) / pi**2 + Rational(1, 2))
 
 
+def test_fourier_series_square_wave():
```

Comment:
Is that name OK? Or should it be `test_FourierSeries_square_wave`? The first one seems to be correct as test uses `fourier_series` rather than `FourierSeries` directly. On the other hand other tests using only `fourier_series` are named with class name.